src/ctap/hid/mod.rs

```diff
@@ -456,14 +380,14 @@ impl CtapHid {
     /// example long user names that are part of the output of an assertion. These cases should be
     /// correctly handled by the CTAP implementation. It is therefore an internal error from the
     /// HID perspective.
-    fn split_message(message: Message) -> HidPacketIterator {
+    pub fn split_message(message: Message) -> HidPacketIterator {
         let cid = message.cid;
         HidPacketIterator::new(message).unwrap_or_else(|| {
             // The error payload is 1 <= 7609 bytes, so unwrap() is safe.
             HidPacketIterator::new(Message {
                 cid,
                 cmd: CtapHidCommand::Cbor,
-                payload: vec![Ctap2StatusCode::CTAP2_ERR_VENDOR_INTERNAL_ERROR as u8],
+                payload: vec![CTAP2_ERR_VENDOR_INTERNAL_ERROR as u8],
```

Comment:
I don't understand this change. What's the motivation behind it?

Reply:
This was the only CTAP dependency and I wanted to make this module independent, so we can later move it around trivially.

... is what I thought, but I just double checked and realized that there is `MAX_MSG_SIZE` as well, that I overlooked. Since that would be a logic change to move it, I reverted the line in question and we can do the move later, if we want.